source/features/locked-issue-indicator.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {LockIcon} from '@primer/octicons-react';
+import * as pageDetect from 'github-url-detection';
+
+import onetime from 'onetime';
+
+import {$, elementExists} from 'select-dom';
+
+import features from '../feature-manager.js';
+import observe from '../helpers/selector-observer.js';
+
+function addLock(element: HTMLElement): void {
+	if (!elementExists('reactions-menu', element)) {
+		// Add locked indicator to header
+		$('.gh-header-meta')!.children[0].after(
+			<div className="flex-shrink-0 mb-2 flex-self-start flex-md-self-center">
+				<span title="Status: Locked" className="State d-flex flex-items-center">
+					<LockIcon className="flex-items-center mr-1"/>
+					Locked
+				</span>
+			</div>,
+		);
+		// Add locked indicator to sticky header
+		$('.gh-header-sticky .flex-row')!.children[0].after(
+			<div className="mr-2 mb-2 flex-shrink-0">
+				<span title="Status: Locked" className="State d-flex flex-items-center">
+					<LockIcon className="flex-items-center mr-1"/>
+					Locked
+				</span>
```

Comment:
This span can be extracted into its own function in this file, e.g. 

```js
const Icon = () => <span>etc</span>
``` 

And then you can use it as `<Icon/>`

Reply:
The positioning was copied from the state label:

![image](https://github.com/refined-github/refined-github/assets/58778985/a148bd83-5c2f-41a4-82f8-c08c61108427)

![image](https://github.com/refined-github/refined-github/assets/58778985/bc31b145-5d54-4f74-a3e7-70944541082a)

Reply:
Discussions also uses this positioning, however they omit both labels in the sticky header

![image](https://github.com/refined-github/refined-github/assets/58778985/3db0868e-69f5-43c3-a366-36d05f9e1984)

![image](https://github.com/refined-github/refined-github/assets/58778985/a4cf1900-580a-4f58-b476-5c7660ae0b4a)